pkg/utils/timeutils.go

```diff
@@ -33,6 +34,14 @@ const (
 	OneDayMs    = 24 * 3600 * 1000
 )
 
+var (
+	// TimeFormats are time formats
+	TimeFormats = []string{
```

Comment:
The comment is a bit redundant I would say 🙂

Reply:
Removed